packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -83,6 +83,7 @@ const NotificationDetail = () => {
   );
 
   const currentRecipient = notification && notification.currentRecipient;
+  const isCancelled = notification.notificationStatus === NotificationStatus.CANCELLED;
```

Comment:
add the check on CANCELLATION_IN_PROGRESS

Reply:
do it